src/test/java/teammates/test/cases/logic/FeedbackResponseCommentsLogicTest.java

```diff
@@ -278,7 +275,7 @@ public void testDeleteFeedbackResponseComment() throws Exception {
         frComment.feedbackResponseId = "invalid responseId";
         //without proper frCommentId and its feedbackResponseId,
```

Comment:
Please delete invalid comment and statement :)

Reply:
yes, removing this.